diff --git a/meson.build b/meson.build
index c8d85f7..cd3dc3b 100644
--- a/meson.build
+++ b/meson.build
@@ -106,8 +106,8 @@ shared_module(
         wingpanel
     ],
     install: true,
-    # install_dir : wingpanel.get_pkgconfig_variable('indicatorsdir'), <- installs to /usr/local/
-    install_dir : '/usr/lib/x86_64-linux-gnu/wingpanel/'
+    install_dir : wingpanel.get_pkgconfig_variable('indicatorsdir'),
+    # install_dir : '/usr/lib/x86_64-linux-gnu/wingpanel/'
 
 )
 
